lib/removePhotoBackground.ts

```diff
@@ -19,18 +20,21 @@ const removeBg = async (fileName, writeFile) => {
   await streamPipeline(response.body, fs.createWriteStream(writeFile));
   return true;
 };
-exports.createImageWithoutBg = async (id) => {
+export default async function removePhotoBackground(id: any) {
```

Comment:
```suggestion
export default async function removePhotoBackground(id: number) {
```
just because it's saved as a int in DB